blocks/condition/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { registerBlockType } from '@wordpress/blocks';
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
+
+import edit from './edit';
+import metadata from './block.json';
+
+import './style.scss';
+
+/* @ts-expect-error Provided types are inaccurate to the actual plugin API. */
+registerBlockType(metadata, {
+  edit,
+  save: () => {
+    const blockProps = useBlockProps.save();
+    return (
+      <div {...blockProps}>
+        {/* @ts-ignore */}
+        <InnerBlocks.Content />
+      </div>
+    );
+  },
```

Comment:
I'm not sure there's much to gain by assigning the `save` function to a variable

```suggestion
  save: () => (
    <div {...useBlockProps.save()}>
      {/* @ts-ignore */}
      <InnerBlocks.Content />
    </div>
  ),
```